cfme/tests/cli/test_appliance_console.py

```diff
@@ -1060,7 +1059,73 @@ def test_appliance_console_network_conf():
             9.
             10. Check IP and hostname values matches in Summary Information.
     """
-    pass
+    appliance = temp_appliance_preconfig_long
+    command_set = ("ap", RETURN)
+    old_result = appliance.appliance_console.run_commands(command_set, timeout=30)
+    logger.info('"ap" command output before test run:%s' % old_result)
+
+    ipv4 = re.search(IPv4_REGEX, old_result[0]).group("ipv4")
+    soft_assert(ipv4)
+    command_set = ("ap", RETURN, app_con_menu["config_net"], "2", ipv4, RETURN, RETURN,
+                   RETURN, RETURN, RETURN, "Y")
+    output = appliance.appliance_console.run_commands(command_set, timeout=40)
+    soft_assert("failed" not in output[-1])
+
+    ipv6 = re.search(IPv6_REGEX, old_result[0]).group("ipv6")
+    soft_assert(ipv6)
+    command_set = ("ap", RETURN, app_con_menu["config_net"], "3", ipv6, RETURN, RETURN,
+                   RETURN, RETURN, RETURN, "Y")
+    output = appliance.appliance_console.run_commands(command_set, timeout=40)
+    soft_assert("failed" not in output[-1])
+
+    hostname = fauxfactory.gen_alphanumeric(start="test", length=20).lower()
+    command_set = ("ap", RETURN, app_con_menu["config_net"], "5", hostname, RETURN)
+    appliance.appliance_console.run_commands(command_set, timeout=30)
+
+    appliance.evmserverd.restart()
+
+    command_set = ("ap", RETURN)
+    new_result = appliance.appliance_console.run_commands(command_set, timeout=30)
+    logger.info('"ap" command output after restart EVM service:%s' % new_result)
+
+    command_set = "ip a"
+    ip_a_ret = appliance.ssh_client.run_command(command_set)
+    soft_assert(ip_a_ret.success)
+    ip_a_output = ip_a_ret.output.split("\n")
+    logger.info('"ip a" command output:%s' % ip_a_output)
```

Comment:
When using the logger, you want to pass the variables to format into the string as arguments to the logger function - not format via `%` all as the first argument.  Small, but its in our coding standards so I'm going to require this for merge :/

Reply:
**Required**

Reply:
Done